tcr-engine/toolchain/gradle_test.go

```diff
@@ -59,7 +59,7 @@ func Test_gradle_toolchain_build_command_path(t *testing.T) {
 }
 
 func Test_gradle_toolchain_build_command_args(t *testing.T) {
-	assertBuildCommandArgs(t, []string{"build", "-x", "test"}, gradleToolchainName)
+	assertBuildCommandArgs(t, []string{"build", "testClasses", "-x", "test"}, gradleToolchainName)
```

Comment:
This looks very similar to what is in gradle.go. I'm wondering if this will create unnecessary test maintenance. I would like to have the minimal checks that it gets the args that are in gradle.go, but without repetition. Can I suggest to test only 1 argument? Or use a constant that we can reuse in both the code and the test?

Reply:
There is indeed duplication between gradle and gradle-wrapper (similarly there is between maven and maven-wrapper) arguments. Let me check what could be done easily to remove these replications

Reply:
In the last commit I've merged gradle and gradle-wrapper toolchains tests to remove duplications. Does this look better to you? If you're ok with it I'll do the same with maven and maven-wrapper tests.